lib/Terminal/Print/Grid.pm6

```diff
@@ -182,6 +182,8 @@ multi method change-cell($x, $y, %c) {
     @!grid[$y][$x] = Cell.new(|%c);
 }
 
+multi method change-cell($x, $y, *%c) { samewith($x, $y, %c) }
```

Comment:
samewith is very slow.  This should probably be an explicit call (and the same with the other one).

Reply:
This can just be removed as well. Just a convenience sugar for being able use plain list of pairs and not wrap it into `{ }`.